src/web/mjs/connectors/MundoMangaKun.mjs

```diff
@@ -0,0 +1,61 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MundoMangaKun extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mundomangakun';
+        super.label = 'Mundo Mangá-Kun';
+        this.tags = [ 'manga', 'portuguese'];
+        this.url = 'https://mundomangakun.com.br';
+    }
+
+    async _getMangaFromURI(uri) {
+        const request = new Request(uri, this.requestOptions);
+        const data = await this.fetchDOM(request, "h1.titulo_projeto");
```

Comment:
Inconsistent double quotes, any other string in this javascript uses single quotes